### PR TITLE
BD-3011: Translation API endpoints EA

### PR DESCRIPTION
**Pull request by lydia-xie**
# Pull Request/Issue Resolution

Closes #**BD-3011**

#### Is this change associated with a Braze feature/product release?
- [ ] Yes (**Insert Feature Release Date Here**)
- [ ] No

---

### CODE REVIEW

**Review by kaiwah — APPROVED** [review]
I just have mostly questions about descriptions, but overall LGTM

File: _docs/_api/endpoints/translations/get_bulk_translations_canvases.md [hunk 1]
```diff
@@ -0,0 +1,107 @@
+---
+nav_title: "GET: Translate Canvas"
```
[the remark this answers is not in the document]
Comment (lydia-xie) [reply _docs/_api/endpoints/translations/get_bulk_translations_canvases.md:2]: Renamed it!

**Comment by djhanggi** [comment]
@kaiwah Refresh my memory, how would a user of the API GET a locale id? I don't see anywhere in the docs mention how to retrieve that

**Comment by kaiwah** [comment]
> @kaiwah Refresh my memory, how would a user of the API GET a locale id? I don't see anywhere in the docs mention how to retrieve that

The locale_id can only be retrieved once you fetch ALL translations

image

File: _docs/_api/endpoints/translations/put_update_translation_canvas.md [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+The following table lists possible returned errors and their associated troubleshooting steps.
+
+| Error message | Troubleshooting |
+| --- | --- |
+|`INVALID_CAMPAIGN_ID`|Confirm the campaign ID matches the campaign you're translating.|
+|`INVALID_LOCALE_ID`|Confirm your locale ID exists in your message translation.|
+|`INVALID_MESSAGE_VARIATION_ID`|Confirm your message ID is correct.|
+|`INVALID_TRANSLATION_OBJECT`|Translation IDs are mismatched or translated text exceeds limits.|
+|`MESSAGE_NOT_FOUND`|Check that the message to be translated.|
+|`LOCALE_NOT_FOUND`|
```
Comment (rachel-feinberg) [inline _docs/_api/endpoints/translations/put_update_translation_canvas.md:94]:
```suggestion
|`LOCALE_NOT_FOUND`| Confirm the locale exists in your multi-language settings. |
```
I pasted this in from put_update_translation_campaign

**Review by rachel-feinberg — APPROVED** [review]
I left a comment for a missing troubleshooting cell. Otherwise, looks good to me!

**Pull request merged**